packages/protocol/test/governance/double_signing_slasher.ts

```diff
@@ -0,0 +1,217 @@
+import { CeloContractName } from '@celo/protocol/lib/registry-utils'
+import { assertRevert, NULL_ADDRESS } from '@celo/protocol/lib/test-utils'
+import { toFixed } from '@celo/utils/lib/fixidity'
+import { addressToPublicKey } from '@celo/utils/lib/signatureUtils'
+import BigNumber from 'bignumber.js'
+import {
+  AccountsContract,
+  AccountsInstance,
+  MockElectionContract,
+  MockElectionInstance,
+  MockLockedGoldContract,
+  MockLockedGoldInstance,
+  RegistryContract,
+  RegistryInstance,
+  TestDoubleSigningSlasherContract,
+  TestDoubleSigningSlasherInstance,
+  ValidatorsTestContract,
+  ValidatorsTestInstance,
+} from 'types'
+
+const Accounts: AccountsContract = artifacts.require('Accounts')
+const Validators: ValidatorsTestContract = artifacts.require('ValidatorsTest')
+const DoubleSigningSlasher: TestDoubleSigningSlasherContract = artifacts.require(
+  'TestDoubleSigningSlasher'
+)
+const MockElection: MockElectionContract = artifacts.require('MockElection')
+const MockLockedGold: MockLockedGoldContract = artifacts.require('MockLockedGold')
+const Registry: RegistryContract = artifacts.require('Registry')
+
+// @ts-ignore
+// TODO(mcortesi): Use BN
+Validators.numberFormat = 'BigNumber'
+// @ts-ignore
+DoubleSigningSlasher.numberFormat = 'BigNumber'
+
+const HOUR = 60 * 60
+const DAY = 24 * HOUR
+// Hard coded in ganache.
+const EPOCH = 100
+
+contract('DoubleSigningSlasher', (accounts: string[]) => {
+  let accountsInstance: AccountsInstance
+  let validators: ValidatorsTestInstance
```

Comment:
Would it be easier to use `MockValidators` for this? Same question for other tests